cmd/qat_plugin/README.md

```diff
@@ -3,26 +3,18 @@
 Table of Contents
 
 * [Introduction](#introduction)
-    * [Modes and Configuration options](#modes-and-configuration-options)
+* [Modes and Configuration options](#modes-and-configuration-options)
 * [Installation](#installation)
     * [Prerequisites](#prerequisites)
-    * [Pre-built image](#pre-built-image)
-    * [Getting the source code:](#getting-the-source-code)
-    * [Deploying as a DaemonSet](#deploying-as-a-daemonset)
-        * [Build the plugin image](#build-the-plugin-image)
-        * [Deploy the DaemonSet](#deploy-the-daemonset)
-        * [Verify QAT device plugin is registered:](#verify-qat-device-plugin-is-registered)
-    * [Deploying by hand](#deploying-by-hand)
-        * [Build QAT device plugin](#build-qat-device-plugin)
-        * [Deploy QAT plugin](#deploy-qat-plugin)
-    * [QAT device plugin Demos](#qat-device-plugin-demos)
-        * [DPDK QAT demos](#dpdk-qat-demos)
-            * [DPDK Prerequisites](#dpdk-prerequisites)
-            * [Build the image](#build-the-image)
-            * [Deploy the pod](#deploy-the-pod)
-            * [Manual test run](#manual-test-run)
-            * [Automated test run](#automated-test-run)
-        * [OpenSSL QAT demo](#openssl-qat-demo)
+    * [Pre-built images](#pre-built-images)
+* [Demos and Testing](#demos-and-testing)
```

Comment:
Link "Verify plugin registration" section is missing

Reply:
fixed in the next rev